doc/install.rst

```diff
@@ -67,7 +67,7 @@ Board Pin    Name   Remarks       RPi Pin   RPi Function
 5            CLK    Clock         23        GPIO 11 (SPI CLK)
 ============ ====== ============= ========= ====================
 
-.. seealso:: See notes section for cascading/daisy-chaining, power supply and
+.. seealso:: Also see the section for :doc:`cascading/daisy-chaining <python-usage>`, power supply and
    level-shifting.
```

Comment:
@rm-hull can i preview this link somewhere? don't have local sphinx available atm

Reply:
https://luma-led-matrix.readthedocs.io/en/thijstriemstra-patch-1/ ... its building at the moment, so its 404ing for me with following, but should be right in a few minutes


        \          SORRY            /
         \                         /
          \    This page does     /
           ]   not exist yet.    [    ,'|
           ]                     [   /  |
           ]___               ___[ ,'   |
           ]  ]\             /[  [ |:   |
           ]  ] \           / [  [ |:   |
           ]  ]  ]         [  [  [ |:   |
           ]  ]  ]__     __[  [  [ |:   |
           ]  ]  ] ]\ _ /[ [  [  [ |:   |
           ]  ]  ] ] (#) [ [  [  [ :===='
           ]  ]  ]_].nHn.[_[  [  [
           ]  ]  ]  HHHHH. [  [  [
           ]  ] /   `HH("N  \ [  [
           ]__]/     HHH  "  \[__[
           ]         NNN         [
           ]         N/"         [
           ]         N H         [
          /          N            \
         /           q,            \
        /                           \

Reply:
thanks! looks good to me now.